### DISCUSSION TITLE
New Type: Visitatori RSA

### DISCUSSION THREAD

**Discussion by marcorocca1973** [DevTrevi/DgcReader#76] [General]
Today in my VerificaC19 I see this new verification type "Visitatori RSA", it's correct?

**Comment by DevTrevi** [comment]
Hi @marcorocca1973 , yes, the latest updates to VerificaC19 includes some new validation types. 
"Visitatori RSA" is replacing the "Booster" mode
You can find more details on [the official app repository](https://github.com/ministero-salute/it-dgc-verificaC19-android) or the [official SDK documentation](https://github.com/ministero-salute/it-dgc-documentation/blob/master/SCANMODE.md)
